docs/tutorial/updates.md

```diff
@@ -261,7 +261,49 @@ server-communication aspect of the process by loading your update from a local d
 
 ## Update server specification
 
-A Squirrel-compatible update server has different
+For advanced deployment needs, you can also roll out your own Squirrel-compatible update server.
```

Comment:
We might want to give people some keywords here on _why_ they might want to roll their own.

E.g. 

* Putting the update server behind an authentication check
* Doing Percentage Based Rollouts
* etc.

Reply:
[90b0b55](https://github.com/electron/electron/pull/43234/commits/90b0b55f82660b3295c70e286ab0768c8cdd0ed9)